### ISSUE TITLE
Stackage nightly test suite failure

### ISSUE THREAD

**Issue by lwm** [diagrams/diagrams-lib#299]
```
test/Test.hs:6:1: error:
    Failed to load interface for ‘Diagrams.Test.Transform.Matrix’
    Perhaps you meant
      Diagrams.Transform.Matrix (from diagrams-lib-1.4.1.1)
    Use -v to see a list of the files searched for.
```

**Comment by byorgey** [comment]
Thanks, I must have forgotten to add this module to other-modules. I'll fix
soon (unless someone else gets to it first).

**Comment by byorgey** [comment]
@lwm released as `diagrams-1.4.1.2`.
